docs/Reference/Server.md

```diff
@@ -1574,6 +1574,21 @@ if (statusCode >= 500) {
   log.error(error)
 }
 ```
+
+⚠️ Avoid calling setErrorHandler multiple times in the same scope.
```

Comment:
This should adhere to the existing warning style:

https://github.com/fastify/fastify/pull/5973